hedera-node/src/main/java/com/hedera/services/queries/answering/StakeAwareAnswerFlow.java

```diff
@@ -0,0 +1,30 @@
+package com.hedera.services.queries.answering;
+
```

Comment:
This class can be deleted if we check as above. If not need a copyright header here